Lombiq.BaseTheme/Services/MainMenuNavigationProvider.cs

```diff
@@ -73,7 +75,9 @@ private async Task AddAsync(NavigationBuilder builder, ContentItem menuItem)
         {
             var nodeList = new HtmlParser().ParseFragment($"<div>{htmlMenuItemPart.Html}</div>", contextElement: null!);
             var textContent = string.Concat(nodeList.Select(x => x.Text()));
-            builder.Add(new LocalizedString(textContent, textContent), menu => menu.Url("#").LocalNav());
+            var url = string.IsNullOrEmpty(htmlMenuItemPart.Url) ? "#" : GetUrl(htmlMenuItemPart.Url);
+
+            builder.Add(new LocalizedString(textContent, textContent), menu => menu.Url(url).LocalNav());
```

Comment:
Even if it's hard to actually localize from a PO file, run `textContent` through the localizer instead of instantiation the string directly.

Reply:
I thought that's hopeless, but sure why not? In fact I already have an idea brewing about this...